spotbugs-tests/src/test/java/edu/umd/cs/findbugs/ba/Issue893Test.java

```diff
@@ -1,21 +1,14 @@
 package edu.umd.cs.findbugs.ba;
 
-import static edu.umd.cs.findbugs.test.CountMatcher.containsExactly;
-import static org.hamcrest.MatcherAssert.assertThat;
-
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-
 import edu.umd.cs.findbugs.AbstractIntegrationTest;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcher;
-import edu.umd.cs.findbugs.test.matcher.BugInstanceMatcherBuilder;
 
 class Issue893Test extends AbstractIntegrationTest {
 
     @Test
     void testIssue() {
-        performAnalysis("ghIssues/Issue893.class");
-        BugInstanceMatcher bugTypeMatcher = new BugInstanceMatcherBuilder().build();
-        assertThat(getBugCollection(), containsExactly(0, bugTypeMatcher));
+        Assertions.assertDoesNotThrow(() -> performAnalysis("ghIssues/Issue893.class"));
```

Comment:
Original issue (https://github.com/spotbugs/spotbugs/issues/893) is about the analysis throwing exception.